x/oracle/types/msgs_test.go

```diff
@@ -73,31 +76,82 @@ func TestMsgAggregateExchangeRateVote(t *testing.T) {
 	exchangeRates := "foo:1.0,bar:1232.123"
 	zeroExchangeRates := "foo:0.0,bar:1232.132"
 	negativeExchangeRates := "foo:-1234.5,bar:1232.132"
-	overFlowExchangeRates := "foo:1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.0,bar:1232.132"
+	overFlowMsgExchangeRates := StringWithCharset(4097, "56432")
+	overFlowExchangeRates := "foo:100000000000000000000000000000000000000000000000000000000000000000000000000000.01,bar:1232.132"
 	validSalt := "0cf33fb528b388660c3a42c3f3250e983395290b75fef255050fb5bc48a6025f"
 	saltWithColon := "0cf33fb528b388660c3a42c3f3250e983395290b75fef255050fb5bc48a6025:"
 	tests := []struct {
-		voter         sdk.AccAddress
+		feeder        sdk.AccAddress
+		validator     sdk.AccAddress
 		salt          string
 		exchangeRates string
 		expectPass    bool
 	}{
-		{addrs[0], validSalt, exchangeRates, true},
-		{addrs[0], validSalt, invalidExchangeRates, false},
-		{addrs[0], validSalt, zeroExchangeRates, false},
-		{addrs[0], validSalt, negativeExchangeRates, false},
-		{addrs[0], validSalt, overFlowExchangeRates, false},
-		{sdk.AccAddress{}, validSalt, exchangeRates, false},
-		{addrs[0], "", exchangeRates, false},
-		{addrs[0], saltWithColon, exchangeRates, false},
+		{addrs[0], addrs[0], validSalt, exchangeRates, true},
```

Comment:
Might want to add some comments on these test cases, this is getting pretty long

Reply:
done at https://github.com/umee-network/umee/pull/798/commits/24c185d170eb74e78796bdc7040b7112716436ec